cdap-unit-test/src/main/java/co/cask/cdap/test/internal/DefaultApplicationManager.java

```diff
@@ -161,6 +161,10 @@ public boolean isRunning(Id.Program programId) {
 
   @Override
   public List<RunRecord> getHistory(Id.Program programId, ProgramRunStatus status) {
-    return appFabricClient.getHistory(programId, status);
+    try {
+      return appFabricClient.getHistory(programId, status);
+    } catch (Exception e) {
+      throw Throwables.propagate(e);
```

Comment:
hmm not possible to propagate a checked exception here? Or is this a pattern in this class?